.github/workflows/codespell.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: CodeSpell
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
We could add
```yml
        with:
          show-progress: false
```
to have less noise in the CI logs.

Reply:
I think it's now the default
![image](https://github.com/rubocop/rubocop-rspec/assets/6916/68bccdc3-b6e4-40ac-a6c2-cac0c371df6b)
![image](https://github.com/rubocop/rubocop-rspec/assets/6916/c541ed92-eaa4-4e75-9266-c847aa17baf9)

Reply:
From https://github.com/rubocop/rubocop-rspec/actions/runs/6076842370/job/16485526462:
<img width="896" alt="image" src="https://github.com/rubocop/rubocop-rspec/assets/22333/3c07e0d9-853a-46fc-9eaa-ef96f78ce92a">
You are correct. But it looks like a bug. Notice how it says `show-progress: true` in the list of run options, and there is no `--progress` flag in the `git` command?

Reply:
I opened an issue: https://github.com/actions/checkout/issues/1453

Reply:
Good catch!